tools/sync_test_server/start_server.sh

```diff
@@ -34,13 +34,16 @@ adb reverse tcp:8888 tcp:8888 || { echo "Failed to reverse adb port." ; exit 1 ;
 # Make sure that Docker works correctly with Github Docker Registry by logging in
 docker login docker.pkg.github.com -u $GITHUB_DOCKER_USER -p $GITHUB_DOCKER_TOKEN
 
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+# Create app configurations
+APP_CONFIG_DIR=`mktemp -d`
+echo $APP_CONFIG_DIR
+$SCRIPTPATH/app_config_generator.sh $APP_CONFIG_DIR $SCRIPTPATH/app_template testapp1-hxsju testapp2-fbxtw
```

Comment:
Any reason for having the actual ids in the naming. I think the pattern should be to place it into a folder with the app name (`testapp1` and `testapp2`) and then `run.sh` of the docker image will write the resulting app id into the `app_id` in the folder. If changing this you would also have to update the `mongodb-realm-command-server.js`. The app names should match those of https://github.com/realm/realm-java/blob/master/realm/realm-library/src/syncTestUtils/kotlin/io/realm/TestApp.kt#L31. 
```suggestion
$SCRIPTPATH/app_config_generator.sh $APP_CONFIG_DIR $SCRIPTPATH/app_template testapp1 testapp2
```